layers/modbustcp.go

```diff
@@ -0,0 +1,138 @@
+// Developed by Vitor Graveto
```

Comment:
Please change to "Copyright 2018, The GoPacket Authors, All rights reserved."

and add your name as an "author" under the top-level AUTHORS file.